client/analytics/report/categories/table.js

```diff
@@ -0,0 +1,175 @@
+		return [
+			{
+				label: __( 'Category', 'wc-admin' ),
+				key: 'category',
+				required: true,
+				isLeftAligned: true,
+				isSortable: true,
+			},
+			{
+				label: __( 'Items sold', 'wc-admin' ),
+				key: 'items_sold',
+				required: true,
+				defaultSort: true,
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'G. Revenue', 'wc-admin' ),
+				key: 'gross_revenue',
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Products', 'wc-admin' ),
+				key: 'products_count',
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Orders', 'wc-admin' ),
+				key: 'orders_count',
+				isSortable: true,
+				isNumeric: true,
+			},
+		];
+	}
+
+	getRowsContent( categories ) {
+		return map( categories, category => {
+			const { category_id, items_sold, gross_revenue, products_count, orders_count } = category;
+
+			// @TODO it should link to the Products report filtered by category
+			const productsLink = (
+				<Link
+					href={ '/analytics/orders?filter=advanced&code_includes=' + category_id }
+					type="wc-admin"
+				>
+					{ products_count }
+				</Link>
+			);
+
+			return [
+				// @TODO it should be the category name, not the category ID
+				{
+					display: category_id,
+					value: category_id,
+				},
+				{
+					display: items_sold,
+					value: items_sold,
+				},
+				{
+					display: formatCurrency( gross_revenue ),
+					value: getCurrencyFormatDecimal( gross_revenue ),
+				},
+				{
+					display: productsLink,
+					value: products_count,
+				},
+				{
+					display: orders_count,
+					value: orders_count,
+				},
+			];
+		} );
+	}
+
+	getSummary( totals ) {
+		if ( ! totals ) {
+			return [];
+		}
+		return [
+			{
+				label: _n( 'category', 'categories', totals.categories_count, 'wc-admin' ),
+				value: totals.categories_count,
+			},
+			{
+				label: _n( 'item sold', 'items sold', totals.items_sold, 'wc-admin' ),
+				value: totals.items_sold,
+			},
+			{
+				label: __( 'gross revenue', 'wc-admin' ),
+				value: formatCurrency( totals.gross_revenue ),
+			},
+			{
+				label: _n( 'orders', 'orders', totals.orders_count, 'wc-admin' ),
+				value: totals.orders_count,
+			},
+		];
+	}
+
+	render() {
+		const { tableData, primaryData } = this.props;
+		const { items, query } = tableData;
+
+		const isError = tableData.isError || primaryData.isError;
+
+		if ( isError ) {
+			return <ReportError isError />;
+		}
+
+		const isRequesting = tableData.isRequesting || primaryData.isRequesting;
+
+		const headers = this.getHeadersContent();
+		const orderedCategories = orderBy( items, query.orderby, query.order );
+		const rows = this.getRowsContent( orderedCategories );
+		const totalRows = get( primaryData, [ 'data', 'totals', 'categories_count' ], items.length );
+		const summary = primaryData.data.totals ? this.getSummary( primaryData.data.totals ) : null;
+
+		return (
+			<TableCard
+				title={ __( 'Coupons', 'wc-admin' ) }
```

Comment:
Should be Categories here and on the next line correct?

Reply:
Right. :sweat_smile: